genie-core/src/main/java/com/netflix/genie/core/services/impl/JobCoordinatorServiceImpl.java

```diff
@@ -134,6 +140,17 @@ public JobCoordinatorServiceImpl(
         this.selectCommandTimer = registry.timer("genie.jobs.submit.localRunner.selectCommand.timer");
         this.selectApplicationsTimer = registry.timer("genie.jobs.submit.localRunner.selectApplications.timer");
         this.setJobEnvironmentTimer = registry.timer("genie.jobs.submit.localRunner.setJobEnvironment.timer");
+
+        if (this.jobsProperties.getUsers().isActiveJobsLimitEnabled()) {
+            final int activeJobsLimit = this.jobsProperties.getUsers().getActiveJobsLimit();
```

Comment:
You definitely shouldn't have to check this every time. You should be able to put `@Size` or other validation in the properties class. You could write your own setter in the properties class as well that should force a quick fail when the property was set to invalid value or that resets the value to something allowed. This checking within the constructor here becomes distributed as you may have this check in multiple places (not that you do now but you could). It should be centralized in the data object itself.